.github/workflows/autogenerated/test-post-with-token.yml

```
@@ -0,0 +1,25 @@
+name: test problem (post-with-token)
+
+on:
+  pull_request:
+    paths:
+      - "packages/problems/post-with-token/**"
+
+jobs:
+  test:
```

actionlint (1.7.4)

job "test" needs job "test-entrypoint" which does not exist in this workflow

(job-needs)